implementations/rust/ockam/ockam_app/src/invitations/commands.rs

```diff
@@ -223,6 +229,13 @@ impl InletDataFromInvitation {
         cli_state: &CliState,
         invitation: &InvitationWithAccess,
     ) -> crate::Result<Option<Self>> {
+        let expires_at =
+            OffsetDateTime::parse(&invitation.invitation.expires_at, &Iso8601::DEFAULT).unwrap();
+        if expires_at > OffsetDateTime::now_utc() {
```

Comment:
```suggestion
        if expires_at < OffsetDateTime::now_utc() {
```

Reply:
Wow, I don't know how to think in dates anymore 😆